whois-rpsl/src/main/java/net/ripe/db/whois/common/rpsl/AttributeSyntax.java

```diff
@@ -69,7 +69,9 @@ public interface AttributeSyntax extends Documented {
             "\n" +
             "An as-set name can also be hierarchical.  A hierarchical set\n" +
             "name is a sequence of set names and AS numbers separated by\n" +
-            "colons \":\".  At least one component of such a name must be\n" +
+            "colons \":\". The first element of the name must be an ASN\n" +
```

Comment:
Replace "ASN" with "AS number" and "finishing" with "ending".

Re-phrase as, `The first element of the name must be an AS number followed a colon and ending with a name`

Reply:
AS number followed **by** a colon?

Reply:
Ah yes thanks! Correct